include/revng/Pipeline/LLVMContainer.h

```diff
@@ -142,49 +143,79 @@ class LLVMContainerBase
     auto BeforeEnumeration = this->enumerate();
```

Comment:
I'd put a `using namespace llvm`

Reply:
i don't think the use is so pervasive that we gain much from doing so